morituri/command/cd.py

```diff
@@ -134,10 +134,26 @@ def do(self):
         if not self.program.metadata:
             # fall back to FreeDB for lookup
             cddbid = self.ittoc.getCDDBValues()
-            cddbmd = self.program.getCDDB(cddbid)
-            if cddbmd:
-                sys.stdout.write('FreeDB identifies disc as %s\n' % cddbmd)
 
+            allow_freedb = getattr(self.options, 'allow_freedb', False)
```

Comment:
there's a default, so I think `allow_freedb = self.options.allow_freedb` is appropriate. Better yet I'd ditch the temporary variable and just use `if self.options.allow_freedb:` on line 147.

Reply:
Good point, I have no idea what pattern I was trying to copy here...